recipes/libxml2/all/conanfile.py

```diff
@@ -235,36 +235,32 @@ def _package_mingw(self):
             if self.options.include_utils:
                 self.run("mingw32-make -f Makefile.mingw install-dist")
 
+    @functools.lru_cache(1)
```

Comment:
nice find

Reply:
I took it from a recipe of @madebr

Reply:
this will become obsolete with conan 2, so enjoy its usage while it lasts :P